Feature/coffee machine

mercurrie opened the pull request:
added main.py (contains main function to run program)
added data.json(contains data used for menu items)

mercurrie committed:
complete coffee maker

added main.py and data.json

mercurrie committed:
complete coffee maker

added main.py and data.json

michaelhousman commented on main.py:
```diff
@@ -0,0 +1,96 @@
+                coins = process_coins()
+                check_transaction(inserted_amount=coins, selection=user_choice, coffee=menu)
+
+    data_file.close()
```
I think you should be able to open the JSON and not have to close it later.

michaelhousman commented on main.py:
```diff
@@ -0,0 +1,96 @@
+    nickels = int(input('How many nickels?: '))
+    pennies = int(input('How many pennies?: '))
+
+    return (quarters * .25) + (dimes * .10) + (nickels * .05) + (pennies * .01)
```
Personally, I would code this as a JSON so the value of each one is at the top of the page, not buried in a function.

michaelhousman commented on main.py:
```diff
@@ -0,0 +1,96 @@
+    if estimated_coffee < 0: print('Sorry there is not enough coffee.')
+    if estimated_water < 0: print('Sorry there is not enough water.')
+
+    return True if estimated_milk > 0 and estimated_coffee > 0 and estimated_water > 0 else False
```
You can simplify this:

`return estimated_milk > 0 and estimated_coffee > 0 and estimated_water > 0`

michaelhousman commented on data.json:
```diff
@@ -0,0 +1,25 @@
+{
```
I don't love the name `data.json`. try to be more descriptive about what's in there.

michaelhousman commented on main.py:
```diff
@@ -0,0 +1,96 @@
+def process_order():
+    data_file = open('data.json')
+    menu = json.load(data_file)
+    on = ''
```
`on` is a weird variable name.  What does this represent?

michaelhousman reviewed (changes requested):
Asked for a few changes (mostly aesthetic)

mercurrie committed:
OOP

changed files to utilize object oriented.
Deleted data.json
included files:
coffee_maker.py
menu.py
money_maker.py
